scripts/main.js

```diff
@@ -71,4 +71,60 @@ const departments = {
     }
 }
 
-console.log(departments);
\ No newline at end of file
+console.log(departments);
+
+// opdracht 1
+console.log("De afdeling Sales heeft " + departments.sales.numberOfEmployees + " medewerkers");
```

Comment:
Dit is een voorbeeld om opdracht 5c te doen
```suggestion
console.log(`De afdeling Sales heeft ${departments.sales.numberOfEmployees} medewerkers`);
```